doc/howto/storage_backup.md

```diff
@@ -0,0 +1,109 @@
+# How to back up storage volumes
+
+There are two ways to back up your storage volumes: with snapshots or with export files.
+Snapshots are cheap and easy to create, while export files are more reliable and can also be used to restore a volume into a different storage pool.
```

Comment:
So just to clarify my original point, as I may not have been clear. I think we should be clearer around the meaning of "cheap" - i.e quick and space efficient. But we also do need to draw attention to the fact that some storage pool types (such as directory and LVM non-thin mode) are not quick or space efficient (so that we don't give users unrealistic expectations).